src/components/maze/GameOverModalContent.tsx

```diff
@@ -58,14 +58,14 @@ export const GameOverModalContent = ({
 
     if (endGameResponse && !endGameResponse.ok) {
       toast({
-        title: 'Error Minting Cheddar',
+        title: 'Failed to add Cheddar to Farmed Balance.',
```

Comment:
I think this text is too long for a title. It might be a better idea to make it shorter and use the description to make it prettyer.

Reply:
Blaze suggested it.
what do you suggest for title and description? I will confirm with him and update.

Reply:
Maybe something like
Title: 'Transaction error' | 'Error farming'
Description: 'Failed to add Cheddar to Farmed Balance.'